control/controlclient/direct.go

```diff
@@ -826,6 +829,16 @@ func (c *Direct) SendUpdate(ctx context.Context) error {
 	return c.sendMapRequest(ctx, false, nil)
 }
 
+// AutoUpdate wraps a bool for naming on the eventbus
```

Comment:
Maybe not in this PR, but this suggests we might want to add a library type at some point, e.g.,

```go
package events

type Value[T any] struct {
    Value T
}
```

so we could write things like `events.Value[time.Time]` (or make aliases based on that). Not something we need to do here, though.